cpp/src/io/parquet/reader_impl.cu

```diff
@@ -643,7 +640,8 @@ class aggregate_reader_metadata {
           // is this legit?
           CUDF_EXPECTS(schema_elem.num_children == 1, "Unexpected number of children for stub");
           auto child_col_name_info = (col_name_info) ? &col_name_info->children[0] : nullptr;
-          return build_column(child_col_name_info, schema_elem.children_idx[0], out_col_array);
+          return build_column(
+            child_col_name_info, schema_elem.children_idx[0], out_col_array, has_list_parent);
```

Comment:
Should `|| col_type == type_id::LIST` be needed from here?

Reply:
No this is a case where we're skipping a meaningless scheme element entirely.  It has no type or other relevance.